### DISCUSSION TITLE
recentmenu.lua 有一键清空功能吗

### DISCUSSION THREAD

**Discussion by AllenCya** [dyphire/mpv-config#108] [Q&A] [answered]
recentmenu.lua 有一键清空功能吗

**Comment by dyphire** [comment, marked as answer]
没有，添加它也不难，但为什么不直接删除 recent.json 文件呢？

**Reply by AllenCya** [reply]
好的，我捣鼓捣鼓

**Reply by AllenCya** [reply]
![image](https://github.com/user-attachments/assets/95df3e57-adf8-4de8-a802-41852a0683d5)
已实现，谢谢！
